persistence/context.go

```diff
@@ -30,13 +31,13 @@ type PostgresContext struct {
 }
 
 func (p PostgresContext) NewSavePoint(bytes []byte) error {
-	log.Println("TODO: NewSavePoint not implemented")
+	p.logger.Info().Bool("TODO", true).Msg("NewSavePoint not implemented")
 	return nil
 }
 
 // TECHDEBT(#327): Guarantee atomicity betweens `prepareBlock`, `insertBlock` and `storeBlock` for save points & rollbacks.
 func (p PostgresContext) RollbackToSavePoint(bytes []byte) error {
-	log.Println("TODO: RollbackToSavePoint not fully implemented")
+	p.logger.Info().Bool("TODO", true).Msg("RollbackToSavePoint not fully implemented")
```

Comment:
NIT: Thoughts on creating a `.TODO` similar to `.Info`?

Reply:
I think that indeed would be useful!

Reply:
It's probably outside of the scope of this ticket I think :)